docs/partitioning.txt

```diff
@@ -22,17 +21,15 @@ Explicit graph partitioning can only enabled against storage backends that suppo
 
 There are two aspects to graph partitioning which can be individually controlled: edge cuts and vertex cuts.
 
-Edge Cut
-~~~~~~~~
+=== Edge Cut
```

Comment:
Edge cuts are more meaningful when the JanusGraph servers are on the same hosts as the storage backend. If you have to make a network call to a different on each hop, the benefit of edge cuts and custom placement strategies can be largely nullified.